handlers/registration.js

```diff
@@ -28,25 +31,55 @@ module.exports.create = async (event, ctx, callback) => {
   }
   const id = parseInt(data.id, 10);
   const eventID = data.eventID;
+  let eventName = "";
   let registrationStatus = data.registrationStatus;
 
   // Check if the event is full
-  if (registrationStatus === 'registered'){
+  if (registrationStatus === 'registered') {
 
     const eventParams = {
       Key: { id: eventID },
       TableName: 'biztechEvents' + process.env.ENVIRONMENT
     }
-    
+
     await docClient.get(eventParams).promise()
-    .then(async(event) => {
-      const counts = await helpers.getEventCounts(eventID)
+      .then(async (event) => {
+        const counts = await helpers.getEventCounts(eventID);
+        eventName = event.Item.ename;
+
+        if (counts.registeredCount >= event.Item.capac) {
+          registrationStatus = 'waitlist'
+        }
+      })
+  }
```

Comment:
Is this in the right spot?

Reply:
i don't remember writing this code - I think Jacques wrote this?

Reply:
Think u messed up the brackets

Reply:
line 54 or? i thought any bracket error would be apparent when i sls deployed and ran the code

Reply:
Ah nvm i see whats happening now my bad